pkg/vmimage/data/etc/yum.repos.d/kickstart.repo

```diff
@@ -7,6 +8,7 @@ sslclientkey=/var/lib/yum/client-key.pem
 enabled=yes
```

Comment:
it might be worth setting enabled=no and then enable them just in time in the scripts. Thanks @mjudeikis  :)